pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -98,11 +98,11 @@ func NewCmdBootstrap(name, fullName string) *cobra.Command {
 		},
 	}
 
-	bootstrapCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "GitOps repository e.g. https://github.com/organisation/repository")
-	bootstrapCmd.Flags().StringVar(&o.GitOpsWebhookSecret, "gitops-webhook-secret", "", "provide the GitHub webhook secret for GitOps repository (if not provided, it will be auto-generated)")
+	bootstrapCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "provide a GitOps repository(it could be GitHub or GitLab) e.g. https://github.com/organisation/repository")
```

Comment:
This probably doesn't really scale all that well, we'll likely provide support for BitBucket, and at that point would we add `it could be GitHub, GitLab or BitBucket` ?

Also, if we have to repeat it on every line, is that useful?

Reply:
made the requested changes